Makefile

```diff
@@ -6,13 +6,16 @@ export CLOVERAGE_VERSION = 1.0.13
 JAVA_VERSION = $(shell lein with-profile +sysutils \
                        sysutils :java-version-simple | cut -d " " -f 2)
 
+test/resources/clojuredocs/export.edn:
+	curl -o $@ https://clojuredocs-edn.netlify.com/export.edn
+
 .inline-deps:
 	lein inline-deps
 	touch .inline-deps
 
 inline-deps: .inline-deps
 
-test: .inline-deps
+test: .inline-deps test/resources/clojuredocs/export.edn
```

Comment:
It's best to put resources under the `cider/nrepl` package for consistency with the existing resources.